app/views/locations/show.html.erb

```diff
@@ -45,3 +45,23 @@
 <p>
   You can also book in person by visiting any Citizens Advice bureau.
 </p>
+
+<% content_for :sidebar do %>
```

Comment:
Any reason for moving the `sidebar` content block out of `layouts/locations.html.erb` and into this view?